client/galaxy/scripts/mvc/library/library-foldertoolbar-view.js

```diff
@@ -1382,12 +1382,12 @@ var FolderToolbarView = Backbone.View.extend({
                 "</label>",
                 "</div>",
                 // create new folder button
-                '<button style="display:none;" data-toggle="tooltip" data-placement="top" title="Create New Folder" class="btn btn-default primary-button toolbtn-create-folder add-library-items toolbar-item" type="button">',
+                '<button style="display:none;" data-toggle="tooltip" data-placement="top" title="Create New Folder" class="btn btn-default primary-button toolbtn-create-folder add-library-items add-library-items-folder toolbar-item" type="button">',
```

Comment:
should we prefix classes used just for testing with something?

Reply:
From Gitter PM:

> I don't really consider them test-only CSS classes - they are useful for people customizing Galaxy's CSS, tour building, automated browser based tooling
The tool form prefixes labels a couple things with a tour_id attribute I think - I'm not a fan of that.
I'm okay with a prefix as long as it captures that this isn't specific to testing or tours etc...